ambry-replication/src/main/java/com.github.ambry.replication/ReplicaThread.java

```diff
@@ -295,7 +295,8 @@ public void run() {
               processReplicaMetadataResponse(missingStoreKeys, replicaMetadataResponseInfo, remoteReplicaInfo,
                   remoteNode);
               ExchangeMetadataResponse exchangeMetadataResponse =
-                  new ExchangeMetadataResponse(missingStoreKeys, replicaMetadataResponseInfo.getFindToken());
+                  new ExchangeMetadataResponse(missingStoreKeys, replicaMetadataResponseInfo.getFindToken(),
+                      replicaMetadataResponseInfo.getRemoteReplicaLagInBytes());
```

Comment:
This is a different case - it comes from the `ReplicaMetadataResponseInfo` which is actually _set_ by the remote server. So the same function means different things to the sender and receiver. I don't think there is a perfect name for this.